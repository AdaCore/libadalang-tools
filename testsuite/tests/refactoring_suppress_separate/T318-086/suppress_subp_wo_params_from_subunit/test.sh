#!/bin/sh
suppress_separate -P default.gpr -S test-bar.adb -L 10 -R 27
