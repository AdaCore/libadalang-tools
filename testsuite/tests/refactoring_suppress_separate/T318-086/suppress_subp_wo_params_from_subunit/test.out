Suppressing <SubpBody ["Bar"] test-bar.adb:10:17-18:9>
test.adb
1:1-1:1 
with Ada.Assertions;
with Ada.Characters.Conversions;
with Ada.Numerics.Elementary_Functions;

3:17-3:30 
   is
      use Ada.Characters.Conversions;
      use Ada.Numerics.Elementary_Functions;

      B : Boolean;
   begin
      Put_Line ("Separate_Do_Nothing");
      Put_Line (Ada.Numerics.Elementary_Functions.Log (10.0)'Image);
      Ada.Assertions.Assert (1 = 1);
      Ada.Assertions.Assert (Is_Character (Wide_Character'('a')));
      B := Baz (2);
   end Bar;


test-bar.adb

